src/app/charts/line-graph.component.ts

```diff
@@ -96,20 +96,23 @@ export class LineGraphComponent {
 
     // Set axis scales
     private setAxisScales(): void {
-        var parseTime = D3.timeParse('%Y-%m-%d');
+        // Time scales only recognize annual and daily data
+        var parseTime = D3.timeParse('%Y'); //D3.timeParse('%Y-%m-%d');
```

Comment:
Yea I thought about a flag but didn't yet because we only have yearly data
It would be ideal if it came through on the indicator API response